src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -45,6 +45,14 @@ var fluid_2_0 = fluid_2_0 || {};
                 listener: "{that}.applier.modelChanged.addListener",
                 args: ["isShowing", "{that}.refreshView"]
             },
+            "onCreate.setAriaControls":{
```

Comment:
According to the ARIA example: http://oaa-accessibility.org/example/20/, not only setting ```role="button"``` for the toggle button, also the sliding panel should have ```role="region"``` set. The example also suggests to set ```aria-labelledby``` on the panel. We probably should consider to add a component option for the panel label.